### PR TITLE
Allow http for foreman in case of unattended installation

### PR DESCRIPTION
**Pull request by iNecas**
And redirect other http calls on Foreman to https version.

### CODE REVIEW

**Comment by iNecas** [comment]
Tests went fine

File: modules/foreman/templates/foreman_https.conf.erb [hunk 1]
```diff
@@ -1,12 +1,11 @@
-<Proxy balancer://foremanthinservers>
+<Proxy balancer://foreman-thin-servers>
   <%- scope.lookupvar("foreman::thin_process_count").to_i.times do |i| -%>
     <%= "BalancerMember http://127.0.0.1:#{scope.lookupvar('foreman::thin_start_port').to_i + i}/#{scope.lookupvar('foreman::deployment_url')}" %>
   <%- end -%>
 </Proxy>
 
 <Location /foreman>
-  RequestHeader set X_FORWARDED_PROTO 'https' env=HTTPS
```
Comment (ares) [inline modules/foreman/templates/foreman_https.conf.erb:8]: Why did you remove this?

Comment (iNecas) [inline modules/foreman/templates/foreman_https.conf.erb:8]: Good point, we really need this one. False assumption on my side. fixed

**Comment by ares** [comment]
ACK
